test/evergreen.yml

```diff
@@ -2939,15 +2980,20 @@ buildvariants:
       PATH=/opt/mongodbtoolchain/v3/bin:$PATH
       CFLAGS="-fsanitize=memory -fno-omit-frame-pointer -fno-optimize-sibling-calls -O1 -ggdb"
     posix_configure_flags:
-      --enable-silent-rules --enable-strict --enable-diagnostic
-      --disable-static --prefix=$(pwd)/LOCAL_INSTALL
+      --enable-silent-rules
```

Comment:
Do we want to enable tcmalloc here?

Reply:
Ideally we should be enabling tcmalloc here, but we get an MSAN leak error when we enable tcmalloc. The issue is described in WT-8195.

Reply:
Ah okay, I see that we will enable it for the build variant as part of WT-8195. Thanks!